samples/Tests/Action.IconUrl.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "This card has an action with an icon. Icon placement and size depend on host.",
+			"wrap": true
+		}
+	],
```

Comment:
Need to run the test app and add all the new expected files.

Reply:
Yeah I've been resolving conflicts in the Test app and working on the new tests